pkg/cloudevents/types/convert.go

```diff
@@ -0,0 +1,248 @@
+package types
```

Comment:
this might not matter so much for 1.0 because the types for extensions are only string, or string encoded.

Reply:
ok I think I am just confused: https://github.com/cloudevents/spec/blob/master/spec.md#extension-context-attributes